pkgs/tools/misc/pridefetch/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, lib, python39, zip }: let
+  version = "1.0.0";
+  sha256 = "sha256-/o4er8bO/3HUFXzP+sC+5DYv9EwmxW05o1RT5fEulEg=";
+
+  name = "pridefetch";
+  src = fetchFromGitHub {
+    owner = "SpyHoodle";
+    repo = name;
+    rev = "v" + version;
+    inherit sha256;
+  };
+in stdenv.mkDerivation {
+  inherit name version;
+  buildInputs = [
+    (python39.withPackages (pythonPackages: with pythonPackages; [
+      distro
+    ]))
+    zip
+  ];
+  unpackPhase = "true";
+  installPhase = ''
+    mkdir -p $out/bin
+    cd ${src}/src
+    zip -r $out/pridefetch.zip *
```

Comment:
I might be tired but this whole installPhase looks kind of strange to me...

Reply:
You're right; I've changed it from all being in installPhase to being partly in installPhase and partly in buildPhase. installPhase now only moves the completed executable to the bin directory, whereas buildPhase zips the files and adds the header so the program can be executed by python